deploy/role.yaml

```diff
@@ -107,3 +107,14 @@ rules:
   - "*"
   verbs:
   - "*"
+- apiGroups:
+  - "batch"
+  resources:
+  - jobs
+  verbs:
+  - get
+  - create
+  - list
+  - delete
+  - watch
+  - update
```

Comment:
Can you run `make yaml` to update the install file?

Reply:
Sure thing!